network-api/networkapi/wagtailpages/templates/wagtailpages/tags/cta/petition.html

```diff
@@ -1,15 +1,18 @@
 <div>
+    <!-- ShareProgress button embedding doc:
+        https://shareprogress-kb-05152019.groovehq.com/help/how-can-i-embed-my-share-buttons-in-my-template
+    -->
 
 	{% if cta.share_twitter %}
-	<div id="share-progress-tw" class='{{cta.share_twitter}} share-progress-button sp_tw_small'></div>
+	<div id="share-progress-tw" class='{{cta.share_twitter}} sp_tw_small d-none'></div>
```

Comment:
Tested it on https://foundation.mozilla.org/en/campaigns/aadhaar/. Seems like the class name `share-progress-button` was only used for SCSS to hide the button... not a class name for generating ShareProgress button.